src/server/main.js

```diff
@@ -1,3 +1,26 @@
-import express from './express';
+import config from './config';
+import express from 'express';
+import {Server} from 'http';
```

Comment:
Let's use Express with http.Server so if one wants to use socket.io, it's already there. No difference in practice to be fair.